ideascube/static/ideascube/vendor/tinymce/langs/readme.md

```diff
@@ -0,0 +1,3 @@
+This is where language files should be placed.
+
+Please DO NOT translate these directly use this service: https://www.transifex.com/projects/p/tinymce/
```

Comment:
Where are the language files? We're going to need the translations as well, aren't we?